libcfnet/file_stream.c

```diff
@@ -592,6 +592,31 @@ bool FileStreamServe(SSL *conn, const char *filename)
 
 #define ERROR_MSG_INTERNAL_CLIENT_ERROR "Internal client error"
 
+
+/**
+ * @brief Get the size of a file
+ *
+ * @param file The file pointer
+ * @return the file size or -1 on error
+ * @note -1 on error is quite handy, because rs_sig_args() iterprets it as
+ *       unknown file size
+ */
+static rs_long_t GetFileSize(FILE *file)
```

Comment:
Do we need to worry about symlinks in this function?

Reply:
Yes. That's why I wanted to suggest `lstat()`, but then I realized it was probably not the desired thing to do. And it made me think how the new protocol handles symlinks. It just copies the contents of the symlink target into a file on the other side, right? Worth a comment.

Reply:
Yup! Added a note in the doc strings :wink: